plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -470,7 +470,7 @@ function wc_get_loop_class() {
 	$loop_index = wc_get_loop_prop( 'loop', 0 );
 	$columns    = absint( max( 1, wc_get_loop_prop( 'columns', wc_get_default_products_per_row() ) ) );
 
-	$loop_index ++;
+	++$loop_index;
```

Comment:
ugh this is auto formatting, forgot to save without. Let me know if you'd prefer me to exclude formatting changes.